app/views/posts/_expanded.html.erb

```diff
@@ -162,9 +162,15 @@
               <% required = required_ids&.include?(tag.id) ? 'is-filled' : '' %>
               <% topic = topic_ids&.include?(tag.id) ? 'is-outlined' : '' %>
               <% moderator = moderator_ids.include?(tag.id) ? 'is-red is-outlined' : '' %>
-              <%= link_to tag.name, tag_path(id: post.category_id, tag_id: tag.id),
-                          class: "badge is-tag #{required} #{topic} #{moderator}",
-                          title: tag.excerpt %>
+              <% if tag.excerpt == "" %>
+                <%= link_to tag.name, tag_path(id: post.category_id, tag_id: tag.id),
+                            class: "badge is-tag #{required} #{topic} #{moderator}",
+                            title: "no description yet" %>
+              <% else %>
+                <%= link_to tag.name, tag_path(id: post.category_id, tag_id: tag.id),
+                            class: "badge is-tag #{required} #{topic} #{moderator}",
+                            title: tag.excerpt %>
+              <% end %>
```

Comment:
Instead of wrapping the whole thing in an if/else, you can just use a ternary:

```ruby
<%= link_to tag.name, tag_path(id: post.category_id, tag_id: tag.id),
                            class: "badge is-tag #{required} #{topic} #{moderator}",
                            title: tag.excerpt.present? ? tag.excerpt : "(no description yet)" %>
```

Same for the other file.